package/MDAnalysis/lib/distances.py

```diff
@@ -206,6 +206,81 @@ def _check_lengths_match(*arrays):
     if not all( a.shape == ref for a in arrays):
         raise ValueError("Input arrays must all be same shape"
                          "Got {0}".format([a.shape for a in arrays]))
+      
+## NOT COMPLETE - NEED REFINEMENT
+def distance(atom1,atom2,box=None):      
+       """
+       Checks the distance of only a pair of atoms, rather than the array check that works with distance_array
+       
+       TODO: refinement of the array structure to take advantage of numpy arrays, and generalize the input var 
+         
+       """
+        name1 = "bynum" + " " + str(atom1) # concatenate string for first selection 
+        name2 = "bynum" + " " + str(atom2) # concatenate string for the second selection 
+        
+        selection1 = u.select_atoms(name1) # select from universe
+        selection2 = u.select_atoms(name2) # ditto          
+ 
+        selection1_vec = selection1.positions[0] # As there is only one atom, we only need to extract the first row 
+        selection2_vec = selection2.positions[0] 
+        
+        selection1_selection2_array =  [selection1_vec, selection2_vec]
+        
+         # --- For a triclinic box --- # 
+        if box is not None:
+            box_type = box_check(box) ## Check type of box and store it in box_type variable
+
+            if box_type == 'tri_box':            
+                for i in range(len(selection1_selection2_array)):
```

Comment:
This looks all a bit complicated. However, **implement your tests** and then see if your solution works. Once you have one solution it can be improved. But tests are important early on.